src/Webinex.Calendar.Tests/EventFilterFactoryTests/EventFilterFactoryScenario.cs

```diff
@@ -184,9 +184,13 @@ public void ToContain(params string[] tags)
 
     private EventRow<TestEventData>[] Filter()
     {
-        var filter = EventFilterFactory.Create<TestEventData>(_period.Start, _period.End!.Value, null, TimeZoneInfo.Utc.Id)
-            .Compile();
-        return _events.SelectMany(x => x.Value).Where(filter).ToArray();
+        var filter = new EventFilters<TestEventData>(
```

Comment:
because inside I do db filtering/client filtering 
![image](https://github.com/webinex/calendar/assets/26519973/e95ec8d1-50d5-41be-a438-4f1e7c8aab3e)

Reply:
yeah, but mb it makes sense to rewrite it to `ToExpression(out var requireClientFiltering)` in case we'd like to use `.AsNoTracking()` (which actually might be added in lots of places) or if we'd like to make grouping